### DISCUSSION TITLE
balance/summary

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#242] [General]
# balance/summary

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/balance/summary.html

**Comment by 8Haixin** [comment]
请问云厂商提供的弹性负载均衡ELB是属于什么类型呀？软件负载均衡吗？

**Reply by isno** [reply]
可以归类到  “云软件”负载均衡。

**Comment by zjyyzhengli** [comment]
”集群还需对外界保持足够的**度透明**“ -- 是”**透明度**“

**Comment by skylar2826** [comment]
文字错误：
4.4 子项： “可高用形式” -> "高可用形式"
存在两个4.5，重复
